frontend/routes/package/(_islands)/DownloadChart.tsx

```diff
@@ -193,7 +193,7 @@ function getSeries(
   ).flat();
 
   const xValues = collectX(
-    dataPointsToDisplay.map((version) => version.downloads).flat(),
+    dataPointsWithDownloads.map((version) => version.downloads).flat(),
```

Comment:
Only this line is the actual fix of the issue. We need to collect x values based on 'all' data, instead of recent '5' versions.